### DISCUSSION TITLE
add objects full duplication

### DISCUSSION THREAD

**Discussion by mushroomeo** [UPBGE/upbge#1498] [Ideas/Features for UPBGE 0.3+]
hi i tested the new added option add objects with full duplications ettings. its very nice for materials but problem is, it doesnt work with constraints . if i add 2 objects and 1 object has rigid body joint constraint to another then rigid body joint constraint will not work somehow.

and cant add objects from instanced collection...

here is the simple blend file to see. 

[constrains duplication not working.zip](https://github.com/UPBGE/upbge/files/6576492/constrains.duplication.not.working.zip)

**Comment by youle31** [comment]
It is not supported. I move this to discussions as it's a more a feature request
